cloudpai/client.go

```diff
@@ -0,0 +1,117 @@
+package cloudpai
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/vektah/gqlparser/v2/gqlerror"
+
+	"time"
+)
+
+// defaultURL for Atlas Cloud.
+const defaultURL = "https://api.atlasgo.cloud/query"
+
+// roundTripper is a http.RoundTripper that adds the Authorization header.
+type roundTripper struct {
+	token string
+}
+
+// RoundTrip implements http.RoundTripper.
+func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Set("Authorization", "Bearer "+r.token)
+	req.Header.Set("User-Agent", "gh-atlas-cli")
```

Comment:
Maybe add a version?

Reply:
I dont want to add build flags at the moment, as it is not necessary yet.